core/src/main/java/org/apache/calcite/avatica/util/AbstractCursor.java

```diff
@@ -173,14 +173,14 @@ protected Accessor createAccessor(ColumnMetaData columnMetaData,
       default:
         throw new AssertionError("bad " + columnMetaData.type.rep);
       }
-    case 2013: // TIME_WITH_TIMEZONE
```

Comment:
Nit:
While these changes are fine, they do not agree with the commit message. (i.e this is not a test refactor)

Reply:
Change the commit message to "[CALCITE-5536](https://issues.apache.org/jira/browse/CALCITE-5536)
Clean up some of the magic numbers in AvaticaResultSetConversionsTest and AbstractCursor" and I think we're good.